tests/builder/subcommands.rs

```diff
@@ -487,20 +487,25 @@ fn bad_multicall_command_error() {
 
     let err = cmd.clone().try_get_matches_from(["world"]).unwrap_err();
     assert_eq!(err.kind(), ErrorKind::InvalidSubcommand);
-    assert_data_eq!(err.to_string(), str![[r#"
+    assert_data_eq!(
+        err.to_string(),
+        str![[r#"
```

Comment:
Looks like your rustfmt disagrees with mine.  If this passes CI, can you move this to its own commit before anything else?

Reply:
It did pass CI, but I'll revert it as I don't it's better than the existing format and I didn't mean to change it. I think `rustfmt` bails on some macro cases.